controllers/glance_controller.go

```diff
@@ -197,6 +197,18 @@ func (r *GlanceReconciler) SetupWithManager(mgr ctrl.Manager) error {
 func (r *GlanceReconciler) reconcileDelete(ctx context.Context, instance *glancev1.Glance, helper *helper.Helper) (ctrl.Result, error) {
 	r.Log.Info(fmt.Sprintf("Reconciling Service '%s' delete", instance.Name))
 
+	// remove db finalizer first
+	db, err := database.GetDatabaseByName(ctx, helper, instance.Name)
```

Comment:
@pinikomarov I see that this comes from [1], so we're basically removing the finalizer checking the db entry.
LGTM, but I guess this change needs rebase first.

[1] https://github.com/openstack-k8s-operators/lib-common/pull/87